tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -3,8 +3,9 @@ import flushPromises from 'flush-promises'
 import { mount, Wrapper } from '@vue/test-utils'
 import CorrectNameRequest from '@/components/correct-name/CorrectNameRequest.vue'
 
-// mock the console.warn function to hide "[Vuetify] Unable to locate target XXX"
-// console.warn = jest.fn()
+// mock the window.alert function, console.warn to hide "[Vuetify] Unable to locate target XXX"
+console.warn = vitest.fn()
+window.alert = vitest.fn()
```

Comment:
Please have a comment for each mock here and describe the warning  you're bypassing.

Reply:
I will